tsconfig.json

```diff
@@ -16,7 +16,8 @@
     "resolveJsonModule": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
-    "experimentalDecorators": true
+    "experimentalDecorators": true,
+    "ignoreDeprecations": "5.0"
```

Comment:
do we need to ignoreDeprecations if tests were upgraded or fixed?

Reply:
yes, tests are only upgraded due to accordion changes in govuk